File: Aizu/01_ITP1_2_A.c
/*
    Problem Code: ITP1_2_A
    Problem Name: Small, Large, or Equal
    Problem Link: https://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=ITP1_2_A
    Description: Write a program which prints small/large/equal relation of given two integers a and b.
*/

#include <stdio.h>

int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    if(a < b){
        printf("a < b\n");
    }
    else if(a > b){
        printf("a > b\n");
    }
    else if(a == b){
        printf("a == b\n");
    }

    return 0;
}

File: Aizu/01_ITP1_2_A.rs
/*
    Problem Code: ITP1_2_A
    Problem Name: Small, Large, or Equal
    Problem Link: https://judge.u-aizu.ac.jp/onlinejudge/description.jsp?id=ITP1_2_A
    Description: Write a program which prints small/large/equal relation of given two integers a and b.
*/

use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let mut iter = input.split_whitespace();
    
    let a: i32 = iter.next().unwrap().parse().unwrap();
    let b: i32 = iter.next().unwrap().parse().unwrap();
    
    if a < b {
        println!("a < b");
    } else if a > b {
        println!("a > b");
    } else {
        println!("a == b");
    }
}
